specs/windows/logical_drives.table

```diff
@@ -2,9 +2,10 @@ table_name("logical_drives")
 description("Details for logical drives on the system. A logical drive generally represents a single partition.")
 schema([
     Column("device_id", TEXT, "The drive id, usually the drive name, e.g., 'C:'."),
-    Column("type", TEXT, "The type of disk drive this logical drive represents."),
-    Column("free_space", BIGINT, "The amount of free space, in bytes, of the drive."),
-    Column("size", BIGINT, "The total amount of space, in bytes, of the drive."),
+    Column("type", TEXT, "Deprecated."),
```

Comment:
nit: Please say this is always going to be unknown.

Reply:
Done!